src/main/kotlin/com/mituuz/fuzzier/StringEvaluator.kt

```diff
@@ -47,10 +47,7 @@ class StringEvaluator(
     }
 
     private fun isExcluded(file: VirtualFile, filePath: String): Boolean {
-        if (changeListManager !== null) {
```

Comment:
I still do think that by default the VSC search should not also use the exclude list. The option could be added to the settings as a separate change.

Having a clear separation and different results to the search results feels to me a bit better. What are your thoughts on this?